### PR TITLE
Rename iTunes Connect to App Store Connect

### PR DESCRIPTION
**Pull request by ofalvai**
### Checklist

- [X] I've read and followed the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- [X] `step.yml` and `README.md` is updated with the changes (if needed)

### Version

Requires a *PATCH* [version update](https://semver.org/)

### Context

iTunes Connect has been renamed to App Store Connect

### CODE REVIEW

**Review by tothszabi — DISMISSED** [review]

File: step.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-title: Deploy to iTunes Connect with Deliver
+title: Deploy to App Store Connect with Deliver
```
Comment (lpusok) [inline step.yml:1]: Should we include `(iTunes Connect)`, so the step can be found using the search in workflow editor?

Comment (ofalvai) [reply step.yml]: Good idea, fixed

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Deploy to iTunes Connect with Deliver
```
Comment (tothszabi) [inline README.md:1]: I have left similar comment on your other PR so I will not go into details and just mark it here as well. 😄

```
Deploy to App Store Connect (formerly iTunes Connect) with Deliver
```

Here and in the `step.yml`.

**Review by tothszabi — APPROVED** [review]
I am happy with it as it is too.

**Pull request merged**